### ISSUE TITLE
Using Web Components in simple Angular Environment not clear

### ISSUE THREAD

**Issue by dimosgit** [SAP/ui5-webcomponents#538]
**Describe the documentation issue**
Documentation is incorrect

**Description of the issue**

Hello, 

I've tried to jump start a very simple Angular app by simply bootstrapping it with Angular CLI. 
1) I've npm install @ui5/webcomponents
2) Tried to use 

`<ui5-button>Hello world!</ui5-button>`

3) Got the error as shown in the picture.

![image](https://user-images.githubusercontent.com/2035725/59269792-83ef0080-8c58-11e9-990c-75d4bcd1a295.png)

So basically there's a missing step in the provided tutorial. This may be a major draw back for a non so experienced developer. 

For those of you who're wondering what's the problem you need to specify that you're using CUSTOM_ELEMENT_SCHEMA as shown in the image bellow:

![image](https://user-images.githubusercontent.com/2035725/59271008-9ae32200-8c5b-11e9-8487-a968288574ba.png)

Regards,
Dimo

**Comment by fifoosid** [comment]
Hi @dimosgit,

In our playground we have described the basic commands for adding UI5 Web Components in your application.

Moreover, every framework you are adding them to, has its own specifics(as the one described in the issue).  Currently we are working on extending the framework documentation. This includes tutorials, on how to init an app, specific to the framework you are using(As we can’t cover all the existing frameworks, we will provide tutorials for the most used ones). Once they are ready, they will be available in the playground.

Best regards,
Filip

**Comment by fifoosid** [comment]
Hi @dimosgit,

The docs are now available in the playground :blush:
